src/meltano/core/tracking/tracker.py

```diff
@@ -24,6 +24,18 @@
 
 CLI_EVENT_SCHEMA = "iglu:com.meltano/cli_event/jsonschema"
 CLI_EVENT_SCHEMA_VERSION = "1-0-0"
+BLOCK_EVENT_SCHEMA = "iglu:com.meltano/block_event/jsonschema"
+BLOCK_EVENT_SCHEMA_VERSION = "1-0-0"
+
+
+class BlockEvents:
+    """Events describing a block state."""
+
+    INITIALIZED: str = "initialized"
+    STARTED: str = "started"
+    COMPLETED: str = "completed"
+    FAILED: str = "failed"
```

Comment:
```suggestion
class BlockEvents(Enum):
    """Events describing a block state."""

    initialized = auto()
    started = auto()
    completed = auto()
    failed = auto()
```

The string versions of each enum member can be retrieved through their `name` attribute.